### PR TITLE
fixed get started section less padding

### LINKED ISSUE #35: GetStarted Component
image

## Requirements:
1. Use `Button` Componet
2. Use `Header` Component
3. Use `Photo` Component

### PR DESCRIPTION
**Pull request by kayliezhao**
fixed the photo component for sizing issues , added the get started section
image

### CODE REVIEW

File: src/components/Photo.tsx [hunk 1]
```diff
@@ -1,22 +1,28 @@
 import React from "react";
 import Image from "next/image";
+import { StaticImageData } from "next/image";
 
 type photoProps = {
   // typescript-react
-  image: string;
+  image: string | StaticImageData;
   width: number;
```
Comment (SafeDuck) [inline src/components/Photo.tsx]: let's make width a string. In particular a tailwind class. 
For example `w-1/2` or `w-1/4`

File: src/components/Photo.tsx [hunk 2]
```diff
@@ -1,22 +1,28 @@
...
-const Photo = ({ image, width, alt }: photoProps) => {
+const Photo = ({ image, alt, width }: photoProps) => {
   return (
-    <div className="relative px-11 pt-10">
-      <Image src={image} alt={alt} width={width} className="opacity-20" />
+    <div className="relative">
       <Image
         src={image}
         alt={alt}
         width={width}
```
Comment (SafeDuck) [inline src/components/Photo.tsx]: Remove the `width` prop from the Image component.
Use a tailwind class instead, aka the string `width` from the props

File: src/components/home/GetStarted.tsx [hunk 3]
```diff
@@ -1,9 +1,44 @@
...
+            <div className="font-archivo mx-11 flex flex-col px-10 pt-10 text-2xl">
+              Just show up and talk to our staff! All we ask is for you to sign
+              our waiver and come ready to work out! For more information check
+              out our “about us”.
+            </div>
+            <div className="mx-11 mt-6 flex flex-col pl-10 pt-10 font-archivo-black text-2xl">
+              Waiver: ***Highlander Gloves Waiver Form (must sign before first
+              practice!)
+            </div>
+            <div className="ml-11 flex w-[200%] translate-x-[-30%] justify-center p-11">
```
Comment (SafeDuck) [inline src/components/home/GetStarted.tsx]: the `translate` shouldn't be needed. This component can be done with `flex` alone

File: src/components/home/GetStarted.tsx [hunk 4]
```diff
@@ -1,9 +1,44 @@
...
+              <Button
+                txt="WAIVER"
+                link="https://docs.google.com/forms/d/1ztQiOcfYzYNZCAQr7l6SpXlNntt2azw3NEqeHk3y8SI/edit"
+              />
+            </div>
+          </div>
+          <div className="ml-11 pr-32">
```
Comment (SafeDuck) [inline src/components/home/GetStarted.tsx]: Too much hard coding padding. 
Use `flex` properties instead. Refer to example:
image

Essentially, we want less on padding and more reliance on `flex` alone

**Review by SafeDuck — CHANGES_REQUESTED** [review]

**Comment by kayliezhao** [comment]
removed width prop, used less padding, adjusted button component so i dont have to translate when i resize the button width
image

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**